web/src/utils/retryImport.ts

```diff
@@ -19,7 +19,7 @@ const retry = async <T>(
 ): Promise<{ default: ComponentType<T> }> => {
   try {
     return componentImport()
-  } catch (error) {
+  } catch (error: any) {
```

Comment:
I'd favor using `unknown` as type annotation, `any` is generally not a recommended type annotation. Same for other usages. We also have a linting rule for this (which is only a warning right now). Maybe we should change that.

Reply:
I had to add `any` explicitly because typescript changed it standard behavior from `any` to `unknown` leading to errors, as properties are not available on unknown, etc.